stable/kuberos/values.yaml

```diff
@@ -62,6 +62,8 @@ image:
 service:
   type: ClusterIP
   port: 80
+  annotations: {}
```

Comment:
can you add this in the readme as well?

Reply:
Done, I also added entries to the existing service fields.